packages/protocol/lib/fixidity.ts

```diff
@@ -0,0 +1,13 @@
+import BigNumber from 'bignumber.js'
```

Comment:
Don't we have an existing library for this?

Reply:
It's moved to the utils package, I believe.